.github/workflows/e2e_on_pull_request.yml

```diff
@@ -25,6 +25,9 @@ jobs:
           - name: K3s-QEMU-SNP
             runner: SNP
             self-hosted: true
+          - name: K3s-QEMU-SNP-GPU
+            runner: SNP
+            self-hosted: true
```

Comment:
Do we need to run this test on every PR? Contention for the 1 GPU could be quite high.

Reply:
I also thought about giving this a more fine-grained trigger. But that's something you and @katexochen will need to decide.

Reply:
Decided in-person with @katexochen to not run this on PRs due to the waiting time.